content/updates/03-2020/3-keep-improving-data-quality.md

```diff
@@ -0,0 +1,27 @@
+# Keep on improving your data quality
+::: meta-data type="Improvement" features="Data Quality Insights, Spell-check, Data Quality" available="5th of March"
+:::
+
+## Data Quality Insights now available for variant products
+
+To help you measure and improve your product data quality, **our Data Quality Insights recommendations tab and grades are now available for your variant products**.
+
+Get specific enrichment and consistency grading scores for all your product variants and quickly identify the list of attributes you need to improve for a specific variant product whether these attributes are at the common or the variants level.
```

Comment:
Do we really talk about the common level?

Reply:
fixed thanks!